charts/ai-service/templates/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.artifactsCache.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "common.fullname" . }}-download-script
+  labels:
+    {{- include "common.labels" . | nindent 4 }}
+data:
+  download-artifacts.sh: |-
+{{ .Files.Get "files/download-artifacts.sh" | indent 4 }}
```

Comment:
💎

Reply:
i see you use another syntax than Konsti in kong-plugins, any reason?
https://github.com/Unique-AG/helm-charts/blob/f133a7a375e263b46c4e76b5f44f461de5a3a2e6/charts/kong-plugins/templates/app-repo-auth.yaml#L6

Reply:
Adapted... both work, but Konsti's is a bit more elegant